packages/codemods/src/codemods/v0.50.x/updateDevFatalErrorPage/updateDevFatalErrorPage.ts

```diff
@@ -36,9 +36,15 @@ export const updateDevFatalErrorPage = async () => {
 
       const text = await res.text()
 
-      const newFatalErrorPage = `${filename}.${isTSPage ? 'tsx' : 'js'}`
-
-      fs.writeFileSync(newFatalErrorPage, text)
+      if (isTSPage) {
```

Comment:
What's the reasoning behind wanting to revert this back if I can ask? Seems natural to have it now update the generated page to .jsx over .js? I know this is a very old version codemod.

Reply:
I had a few other codemods show up in a search and I *didn't* convert them, since they were from old revisions. I figured they would be using Josh's new codemod to upgrade *everything* to JSX...

Reply:
Okay, that's true. The route would be run all old codemods then the new one will do the conversation. Sorry about that then!

Reply:
Sweet, removed!